packages/app/src/elements/import-dialog.ts

```diff
@@ -12,6 +12,13 @@ import { html } from "lit";
 import { saveFile } from "@padloc/core/src/platform";
 import { stringToBytes } from "@padloc/core/src/encoding";
 
+const fieldTypeOptions = Object.keys(FIELD_DEFS).map((fieldType) => ({
+    name: FIELD_DEFS[fieldType].name as string,
+    value: fieldType,
+}));
+// TODO: Remove this
+console.log(fieldTypeOptions);
+
 @customElement("pl-import-dialog")
```

Comment:
@MaKleSoft is there a better UI element to use for this? If I'm limited to this dialog's "width", I'll list the columns vertically (which will also work for however many columns there are, instead of a horizontal scroll), though it's less familiar (visually). Also, I couldn't find any other UX pattern with such a complex/large dialog, and am not sure we should introduce one.

Reply:
@MaKleSoft in complexity I meant in terms of layering (one dialog -> import -> column mapping). Those examples are still helpful, though!